defaultErrorPages/views/validatorErrors.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='utf-8'>
+    <meta http-equiv='Content-Type' content='text/html; charset=utf-8'>
+    <!--[if IE]><meta http-equiv='X-UA-Compatible' content='IE=edge'><![endif]--> {! fix for IE9 rendering in quirks mode !}
+    <meta name='viewport' content='width=device-width,initial-scale=1'>
+    <meta name='format-detection' content='telephone=no'>
+    <title>{pageTitle}</title>
+  </head>
+  <body>
+    <main>
+      <article>
+        <style>h1 {color: #000000;} body{color: #FF0000; margin-left: 15px;}</style>
```

Comment:
change this to `<style>h1 {color: #000;} body {color: #f00; margin-left: 15px;}</style>`